s/shellcheck

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o errexit
+set -o nounset
+
+
+for f in $(git ls-files); do
+  if grep -Eq '^#!(.*/|.*env +)(sh|bash|ksh)' "$f"; then
```

Comment:
can you add a comment explaining what this is intended to match on?

Does this handle files with only a shebang or files with only a `.sh` extension?